examples/config-file/simple-configmap.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: simple-configmap
+  namespace: zarf
+data:
+  templateme.properties: |
+    zebra=###ZARF_VAR_DEPLOY_ZEBRA###
```

Comment:
Recommend changing the variable name. It confused me for a minute thinking `ZARF_VAR_DEPLOY` was the necessary prerequisite vs just `ZARF_VAR`.

Maybe different animals for the deploy vars?

Reply:
le fixed